### ISSUE TITLE
Override NonNullType by Nullable

### ISSUE THREAD

**Issue by Fi1osof** [Urigo/merge-graphql-schemas#184]
Base schema:
```
input ResourceCreateInput {
  alias: String!
}
```

Extends by:
```
input ResourceCreateInput {
  alias: String
}
```

Got error
```
Cannot read property 'value' of undefined
```

**Comment by ozanmuyes** [comment]
Just out of curiosity why would you want to do this? I mean what's the motivation behind the wish/necessity to override an input type?

I think the input definition should be used by a particular mutation, which has a corresponding resolver function belonging a feature/module. If you override it later somewhere else I think it should be the one after merging in the first place.

**Comment by Fi1osof** [comment]
@ozanmuyes i got 3-rdparty component wich allow set empty some fields. I would not wont to override resolver with not necessery checking. Schema will did this.
Second reason: I can set request: 
```
mutation {
  updateUser(
    where: {
      id: "userId"
    }
    data: {
      Topic: {
        update: {
          data: {
            notEmptyField: null,
          }
        }
      }
    }
  ){
    ...user
  }
}
```

Will be used not direct resolver.

**Comment by ozanmuyes** [comment]
Thanks for the explanation, even though I was unable to wrap my head around it :) Hope this bug gets fixed.

**Comment by kamilkisiela** [comment]
Should be fixed by https://github.com/ardatan/graphql-toolkit/pull/189

**Comment by kamilkisiela** [comment]
Fixed with `v1.7.0`
